tests/e2e/utils.go

```diff
@@ -77,15 +77,17 @@ func tlsInfo(t testing.TB, cfg e2e.ClientConfig) (*transport.TLSInfo, error) {
 	}
 }
 
-func fillEtcdWithData(ctx context.Context, c *clientv3.Client, keyCount int, valueSize uint) error {
+func fillEtcdWithData(ctx context.Context, c *clientv3.Client, dbSize int) error {
```

Comment:
Why this change ? 

For different tests the number of keys and size of values might be more important than overall size. 
If you need a simpler api, I would propose a wrapping function: 

`fillEtcdWithDataTotal(...)`

or 
`fillEtcdWithData(..., dbSize)` calls
`fillEtcdWithKeyValues(..., keyCount, keySize, valueSize)`

Reply:
I have tested it, keys have some impact but not enough to expose it now. I think exposing etcd gives better understanding of intention.

Reply:
[I assume this file is a generic testing infrastructure and different tests might have a different needs]

Reply:
Sure, however let's not design something for future that can be easily changed and might never be useful.